examples/terraform/azure/variables.tf

```diff
@@ -270,3 +270,9 @@ variable "rhsm_offline_token" {
   type        = string
   sensitive   = true
 }
+
+variable "disable_auto_update" {
+  description = "Flatcar specific"
```

Comment:
It's obvious from the name, but let's add a proper description so it renders well in docs.

Reply:
Done